spec/helpers/posts_helper_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
```

Comment:
The rails_helper.rb file contains the includes so I can use Devise and FactoryGirl methods. Is there a better place to put those?